GirafIntegrationTest/tests/test_account_controller.py

```diff
@@ -63,6 +63,32 @@ def test_account_can_get_guardian_id(self):
         self.assertIsNotNone(response['data']['id'])
         guardian_id = response['data']['id']
 
+    @order
+    def test_account_cannot_register_citizen_without_displayName(self):
+        """
+        Testing registering a citizen fails without displayName
+
+        Endpoint: POST:/v1/Account/register
+        """
+        data = {'username': 'myUsername', 'password': 'password',
+                'role': 'Citizen', 'departmentId': 1}
+        response = post(f'{BASE_URL}v1/Account/register', json=data, headers=auth(guardian_token)).json()
+        self.assertFalse(response['success'])
+        self.assertEqual(response['errorKey'], 'MissingProperties')
+
+    @order
+    def test_account_cannot_register_citizen_with_empty_displayName(self):
+        """
+        Testing registering a citizen fails with empty displayName
+
+        Endpoint: POST:/v1/Account/register
+        """
+        data = {'username': 'myUsername', 'displayName': '', 'password': 'password',
+                'role': 'Citizen', 'departmentId': 1}
+        response = post(f'{BASE_URL}v1/Account/register', json=data, headers=auth(guardian_token)).json()
+        self.assertFalse(response['success'])
+        self.assertEqual(response['errorKey'], 'MissingProperties')
+
```

Comment:
The HTTP codes are being updated with #61 currently ([here](https://github.com/aau-giraf/web-api/tree/fixes_61/GirafIntegrationTest/tests)), and with that the integration tests are being changed to include checks on response codes. E.g. https://github.com/aau-giraf/web-api/blob/4d7cf72da7df29a56caf3cf4796c5fa3d456a702/GirafIntegrationTest/tests/test_account_controller.py#L54-L71

Your (new) tests should probably conform to this, although I'm not sure if this is your responsibility or not.

Reply:
As for now this functionality is not implemented.
Therefore done "the old way", until the functionality (i.e. the responses return an error code instead) is created. At that done / when that branch is done it will be rebased and include this test which they should then adapt.